src/backend/mailbox_manager/management/commands/setup_dimail_db.py

```diff
@@ -62,17 +62,19 @@ def handle(self, *args, **options):
         # we create a dimail user for keycloak+regie user John Doe
         # This way, la Régie will be able to make request in the name of
         # this user
-        people_base_user = User.objects.filter(name="John Doe")
-        if people_base_user.exists():
-            self.create_user(name=people_base_user.sub, password="whatever")  # noqa S106
+        try:
+            people_base_user = User.objects.get(email="people@people.world")
+        except User.DoesNotExist:
+            self.stdout.write("people@people.world user not found", ending="\n")
+        else:
+            self.create_user(name=people_base_user.sub)
             self.create_allows(people_base_user.sub, domain_name)
-
-        self.stdout.write("DONE", ending="\n")
+            self.stdout.write("DONE", ending="\n")
```

Comment:
Don't need to change indentation here ;) the "done" regards the whole execution, not only the user creation.